tests/integration/targets/azure_rm_sqlserver/tasks/main.yml

```diff
@@ -47,6 +49,108 @@
       - output.changed == false
       - output.state == 'Ready'
 
+- name: Create extended instance of SQL Server
+  azure_rm_sqlserver:
+    resource_group: "{{ resource_group }}"
+    name: "sqlsrv-extended-{{ random_postfix }}"
+    location: eastus
+    admin_username: mylogin2
+    admin_password: Password123!
+    minimal_tls_version: '1.2'
+    public_network_access: Disabled
+    restrict_outbound_network_access: Enabled
+    tags:
+      aaa: bbb
+  register: output
+- name: Assert the resource instance is well created
+  assert:
+    that:
+      - output.changed
+      - output.state == 'Ready'
+
+- name: Create extended instance of SQL Server - idempotent
+  azure_rm_sqlserver:
+    resource_group: "{{ resource_group }}"
+    name: "sqlsrv-extended-{{ random_postfix }}"
+    location: eastus
+    admin_username: mylogin2
+    admin_password: Password123!
+    minimal_tls_version: '1.2'
+    public_network_access: Disabled
+    restrict_outbound_network_access: Enabled
+    tags:
+      aaa: bbb
+  register: output
+- name: Assert the state has not changed
+  assert:
+    that:
+      - not output.changed
+      - output.state == 'Ready'
+
+- name: Update SQL admin password
+  azure_rm_sqlserver:
+    resource_group: "{{ resource_group }}"
+    name: "sqlsrv-extended-{{ random_postfix }}"
+    location: eastus
+    admin_password: Password123!321!
+    change_admin_password: true
+  register: output
+- name: Assert the state has changed
+  assert:
+    that:
+      - output.changed
+
+- name: Get Azure AD group info
```

Comment:
Because this use case uses our pipeline environment but does not have permissions to operate azure_rm_adgroup *. You are advised to delete it. Thank you very much!

Reply:
My apologies, @Fred-sun! I thought that your pipeline just restricted creation of groups and not also retrieving them.

I have updated to remove azure_rm_adgroup* tasks. Please check again. Thank you!